Sources/App/Features/Ticket Hub/Controllers/TicketHubRouteController.swift

```diff
@@ -0,0 +1,310 @@
+import Fluent
+import Vapor
+
+struct TicketHubRouteController: RouteCollection {
+    func boot(routes: RoutesBuilder) throws {
+        routes.grouped(ValidTicketMiddleware()).group("ticket") { builder in
+            builder.get { req -> View in
+                guard let currentEvent = try await Event.query(on: req.db).filter(\.$name == "SwiftLeeds 2023").first() else {
+                // guard let currentEvent = req.storage.get(CurrentEventKey.self) else {
+                    throw Abort(.badRequest, reason: "unable to identify current event")
+                }
+                
+                // 30 days refund window
+                let refundDeadline = Date(timeIntervalSince1970: currentEvent.date.timeIntervalSince1970 - (60 * 60 * 24 * 30))
```

Comment:
@adamrushy - this denotes a 30 day refund window. Please let me know if this is wrong.

Reply:
Can we put that value into an environment settings?  Just in case it needs to change at any point

Reply:
Sure, but would you mind if I pulled this out into the next ticket? I have a piece of work to restructure how we do environment variables to reduce costs.